frontend/appflowy_flutter/lib/workspace/application/settings/appearance/base_appearance.dart

```diff
@@ -31,8 +30,7 @@ abstract class BaseAppearance {
     double? lineHeight,
   }) {
     fontSize = fontSize ?? FontSizes.s14;
-    fontWeight = fontWeight ??
```

Comment:
Is it a request from the designer?

Reply:
Not explicitly, no. But ever since moving from Poppins, the mockups have been mostly regular, and use medium to symbolize importance

Reply:
Makes sense right, the normal body fontweight is regular.

But I'm wondering how this affects text that already go by the original case, do we need to double check everywhere?

Reply:
Yeah... that's the thing. I'm checking as much as I can

Reply:
There are some texts in buttons that may be affected. I remember that most text font weights in buttons should be 500.

Reply:
Done! It defaults to w500 nicely now. See the latest commit